### PR TITLE
feat: upgrade to nutmeg

### PR DESCRIPTION
**Pull request by ghassanmas**
This upgrades to nutmug, luckily there was no issues to resolve. 
Testing with nutmeg openedx and learning MFE, forums behaved normally. 
Also checking the configuration and cs_comments repos I didn't see/noticed any upgrade related that would cause to change the Dockerfile, e.g. edx.org still uses ruby 2.5.

### CODE REVIEW

File: tutorforum/plugin.py [hunk 1]
```diff
@@ -18,22 +17,47 @@
...
+        pkg_resources.resource_filename("tutorforum", "patches"),
+        "*",
+    )
+):
+    with open(path, encoding="utf-8") as patch_file:
+        tutor_hooks.Filters.ENV_PATCHES.add_item((os.path.basename(path), patch_file.read()))
+# Add configuration entries
+tutor_hooks.Filters.CONFIG_DEFAULTS.add_items(
+    [
+        (f"YOUR_PLUGIN_{key}", value)
```
Comment (regisb) [inline tutorforum/plugin.py:59]: @ghassanmas you forgot to replace " YOUR_PLUGIN" by "FORUM". As a consequence the nutmeg deployment is broken: https://github.com/overhangio/nutmeg-demo/runs/6751491100?check_suite_focus=true
Can you please fix this PR such that I can re-trigger a deploy?

Comment (ghassanmas) [reply tutorforum/plugin.py]: oh snap!, I just pushed the changes. Though I had built the image the run/test it locally. I knew I would better of make the nutmeg use a commit hash before I upgrdae to v1.

File: tutorforum/plugin.py [hunk 2]
```diff
@@ -18,22 +17,47 @@
...
-    return all_patches
+# Add the "templates" folder as a template root
+tutor_hooks.Filters.ENV_TEMPLATE_ROOTS.add_item(
+    pkg_resources.resource_filename("tutorforum", "templates")
+)
+# Render the "build" and "apps" folders
+tutor_hooks.Filters.ENV_TEMPLATE_TARGETS.add_items(
+    [
+        ("tutorforum/build", "plugins"),
+        ("tutorforum/apps", "plugins"),
```
Comment (regisb) [inline tutorforum/plugin.py:44]: This is incorrect. It should be `("forum/build", ...)`. As a consequence the forum image build is broken : https://github.com/overhangio/nutmeg-demo/runs/6752404173?check_suite_focus=true
Please delete your tutor environment and test your changes!

Comment (ghassanmas) [reply tutorforum/plugin.py]: I built locally two times, also while reinstalling tutor and tutor fourm as well.

Comment (ghassanmas) [reply tutorforum/plugin.py]: Also at one instance I had simlair error after applying your suggested fixes, and I just resolved by running "tutor config save"

Comment (ghassanmas) [reply tutorforum/plugin.py]: I was able to reproduce and the suggested changes did fix build the error.

**Pull request merged**